go/vt/sqlparser/ast.go

```diff
@@ -177,11 +177,12 @@ type (
 
 	// DBDDL represents a CREATE, DROP, or ALTER database statement.
 	DBDDL struct {
-		Action   string
-		DBName   string
-		IfExists bool
-		Collate  string
-		Charset  string
+		Action      string
+		DBName      string
+		IfExists    BoolVal
```

Comment:
Hmm... I think `BoolVal` is an `Expr`, and that is not what we need here, right? I would suggest we use a good old `bool` instead, like ifExists was doing earlier.